dbActions.py

```diff
@@ -0,0 +1,71 @@
+import sqlite3
+import Settings
+
+
+def create_table(con):
```

Comment:
I would probably choose other name than "con". maybe db_connection would be better one.